src/AWS.Deploy.Orchestration/SystemCapabilities.cs

```diff
@@ -27,4 +27,30 @@ public class DockerInfo
             DockerContainerType = dockerContainerType;
         }
     }
+
+    public class SystemCapability
+    {
+        public string Name {  get; set; }
+        public bool Installed { get; set; }
+        public bool Available { get; set; }
+        public string? Message { get; set; }
+        public string? InstallationUrl { get; set; }
+
+        public SystemCapability(string name, bool installed, bool available)
+        {
+            Name = name;
+            Installed = installed;
+            Available = available;
+        }
+
+        public override string ToString()
+        {
+            if (!string.IsNullOrEmpty(Message))
+                return Message;
```

Comment:
is it expected to ignore all properties from ToString when Message is not null?

Reply:
I changed to a dedicated function instead of overriding ToString and avoid any unwanted behavior.